### PR TITLE
[plugin.video.embycon] 1.9.25

### PR DESCRIPTION
**Pull request by faush01**
### Description

Fixes for the upcoming Emby Server 4.2 release:
-fix the auto resume on playback code
-fix the search, use new search api
-fix an issue with clearing cached data on startup
-allow overriding some video codecs to force transcode

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.embycon/resources/lib/play_utils.py [hunk 1]
```diff
@@ -356,37 +356,59 @@ def playFile(play_info, monitor):
...
-    xbmc.Player().play(playlist)
+    player.play(playlist)
+
+    if seekTime != 0:
+        player.pause()
+        monitor = xbmc.Monitor()
+        count = 0
+        while not player.isPlaying() and not monitor.abortRequested() and count != 100:
+            count = count + 1
+            xbmc.sleep(100)
```
Comment (enen92) [inline plugin.video.embycon/resources/lib/play_utils.py]: you might want to use waitforabort(1) here and use 10 in the count variable

Comment (faush01) [reply plugin.video.embycon/resources/lib/play_utils.py]: yeah I did try that but there was a very noticeable delay from when the play stared to when the seek happened. This approach worked better in testing on various hardware.

**Comment by Rechi** [comment]
@faush01 this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Comment by faush01** [comment]
> 
> 
> @faush01 this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
> Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

Would love to start working on a Python 3 version but still waiting for this PR though:

https://github.com/xbmc/xbmc/pull/16116

**Pull request merged**